cmd/util/flags/configflags/libp2p.go

```diff
@@ -10,11 +10,17 @@ var Libp2pFlags = []Definition{
 		Description: `A comma-separated list of libp2p multiaddress to connect to. ` +
 			`Use "none" to avoid connecting to any peer, ` +
 			`"env" to connect to the default peer list of your active environment (see BACALHAU_ENVIRONMENT env var).`,
+		Deprecated: true,
+		DeprecatedMessage: "The libp2p transport will be deprecated in a future version in favour of using " +
+			"--orchestrators to specify a requester node to connect to.",
 	},
 	{
 		FlagName:     "swarm-port",
 		ConfigPath:   types.NodeLibp2pSwarmPort,
 		DefaultValue: Default.Node.Libp2p.SwarmPort,
 		Description:  `The port to listen on for swarm connections.`,
+		Deprecated:   true,
+		DeprecatedMessage: "The libp2p transport will be deprecated in a future version in favour of using " +
```

Comment:
isn't it better to not have a line break delimiter? E.g. use ``? Just coding style i guess.

Reply:
Yeah probably should have done that. Adding this line break (`+`) is what my editor does when I go over the row length limit.